### ISSUE TITLE
small mistake in the docs

### ISSUE THREAD

**Issue by vesper8** [jdavidbakr/mail-tracker#28]
In the part of the docs where you explain how to *Passing data to the event listeners*

this block

```
public function handle(EmailSentEvent $event)
{
    $tracker = $event->sent_email;
    $model_id = $event->getHeader('X-Model-ID');
    $model = Model::find($model_id);
    // Perform your tracking/linking tasks on $model knowing the SentEmail object
}
```

should be

```
    public function handle(EmailSentEvent $event)
    {
        $tracker = $event->sent_email;
        $model_id = $event->sent_email->getHeader('X-Model-ID');
        $model = Model::find($model_id);
        // Perform your tracking/linking tasks on $model knowing the SentEmail object
    }
```

Otherwise this works great!

**Comment by NathanGeerinck** [comment]
Indeed, I've got the same issue when I wanted to use this.

**Comment by jdavidbakr** [comment]
Fixed in 2.1.6
